ssh.go

```diff
@@ -74,6 +75,13 @@ func (t *sshSmartSubtransport) Action(urlString string, action SmartServiceActio
 		return nil, err
 	}
 
+	// Escape \ and '.
+	uPath := strings.ReplaceAll(u.Path, `\`, `\\`)
+	uPath = strings.ReplaceAll(uPath, `'`, `\'`)
```

Comment:
```suggestion
	uPath := strings.Replace(u.Path, `\`, `\\`, -1)
	uPath = strings.Replace(uPath, `'`, `\'`, -1)
```

ugh, i just realized this is not there yet in Go 1.11. Since this is easy to support in older versions of Go (thanks to [`strings.ReplaceAll`](https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/strings/strings.go;l=963) being a very thin wrapper around `strings.Replace`), let's make this change.